### DISCUSSION TITLE
Welcome to umami Discussions!

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#578] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by heymage** [comment]
Hey @mikecao, thank you so much for umami! Super nice tool 👍🏻

**Comment by mehdiKhenissi** [comment]
hi,
hope every one doing good.
i'm trying to integrate umami in php and i have problem finding where i should pass Cookie umami.auth value ( header or ....) 
can someone help please ?
